test/python/pulse/test_block.py

```diff
@@ -429,6 +430,25 @@ def test_inherit_from(self):
         self.assertEqual(new_sched.name, ref_name)
         self.assertDictEqual(new_sched.metadata, ref_metadata)
 
+    @unittest.skipUnless(has_aer(), "qiskit-aer doesn't appear to be installed.")
+    def test_execute_block(self):
+        """Test executing a ScheduleBlock on a Pulse backend"""
+
+        ref_header = {"memory_slots": 1, "name": "test_block", "metadata": {}, "ode_t": 0.0}
+        ref_data = dict()
+        ref_data["counts"] = {"0x0": 1024}
+        ref_data["statevector"] = [[1.0, 0.0], [0.0, 0.0], [0.0, 0.0]]
```

Comment:
The statevector is dependent on the number of qubits in the backend - it would be good to just read this out from `self.backend`, rather than hard-coding it, so it's easier to manage changes if the backend changes.

Also, why does this statevector not have a power of two number of elements?

Reply:
I am not really familiar with Pulse level schedules as of now so I may not be able to answer that correctly but what this seems to me is that the state of a **single qubit** (thus a 2-dimensional vector) evolving in different time steps (again, not sure ([0 0] may mean qubit has decohered!) )
As of now, I have changed the test according to Matthew's comment below and thus, this is not being tested ATM.